### DISCUSSION TITLE
64bit DBeaver + 32 ODBC drivers

### DISCUSSION THREAD

**Discussion by RWagner2021** [dbeaver/dbeaver#10910] [Q&A] [answered]
I have to use a 32-bit ODBC driver on a 64-bit Windows 10 machine. As I understand things, DBeaver Community Edition 7.3.2 is now only available as a 64-bit application. According to [this old post](https://dbeaver.io/forum/viewtopic.php?f=2&t=789) that should not be a problem. The solution is to install 32-bit Java and configure DBeaver to use Java's JRE. My confusion is which Java product to install.

The Java notes on the [Download page](https://dbeaver.io/download/) mentions that Java 11 or higher is needed. But the latest version of Java is 8 (jre-8u271-windows-i586.exe). What should I do? Thanks.

**Comment by RWagner2021** [comment]
So much for that idea. I found [this post](https://github.com/dbeaver/dbeaver/issues/5678) where is states that DBeaver stopped working with 32-bit Java since 6.0.1.

**Comment by p91paul** [comment, marked as answer]
You can try installing an old enough eclipse version, and then dbeaver as an eclipse plugin instead of the standalone install.

**Comment by RWagner2021** [comment]
Thanks. That's good to know. Since I have to use an older version of something I will first try the standalone installer of DBeaver 6.0.0.
